mobile_back_tflite/cpp/backend_tflite/neuron/tflite_settings_mtk.h

```diff
@@ -41,7 +41,7 @@ benchmark_setting {
   benchmark_id: "image_classification_offline"
   accelerator: "neuron"
```

Comment:
The `accelerator` field should describe the hardware. @freedomtan Can you check if this value is correct?